tests/test_frame.py

```diff
@@ -1087,6 +1087,16 @@ def test_types_pivot_table() -> None:
         pd.DataFrame,
     )
 
+    def test_pivot_table_sort():
+        df = DataFrame(
+            {'a': [1, 2],
+             'b': [3, 4],
+             'c': [5, 6],
+             'd': [7, 8]}
+        )
+        check(assert_type(df.pivot_table(values='a', index='b', columns='c', sort=True),
```

Comment:
You need to use double quotes here.

Be sure to follow the instructions at https://github.com/pandas-dev/pandas-stubs/blob/main/docs/setup.md to set up your environment.  And run tests as shown here:  https://github.com/pandas-dev/pandas-stubs/blob/main/docs/tests.md